api/schemas/social_links.py

```diff
@@ -0,0 +1,19 @@
+from pydantic import BaseModel
```

Comment:
Should we be using Url types from pydantic for this?
```suggestion
from pydantic import BaseModel, AnyHttpUrl
```